dev/DynamicDependency/Powershell/MsixDynamicDependency.psm1

```diff
@@ -0,0 +1,341 @@
+# Copyright (c) Microsoft Corporation and Contributors.
+# Licensed under the MIT License.
+
+<#
+.SYNOPSIS
+Functions to use Windows' Dynamic Dependency API.
+
+.DESCRIPTION
+TBD
+
+.INPUTS
+None
+
+.OUTPUTS
+None
+
+.NOTES
+    Version : 0.1.0
+#>
+
+Add-Type -TypeDefinition @"
```

Comment:
The following blob must match a DynDep header somewhere?

Reply:
Yup. Prototypes and constants as per appmodel.h and doc'd on microsoft.com.